Polyhedron/demo/Polyhedron/Use_ssh.cpp

```diff
@@ -143,6 +144,7 @@ bool establish_ssh_session_from_agent(ssh_session& session,
                                       const char *server,
                                       const char *pub_key_path)
 {
+#ifndef _WIN32
```

Comment:
What is the issue with libssh and Windows? Why is it limited to that function `establish_ssh_session_from_agent`?

Reply:
in libssh.h,there is a #ifndef _WIN32 around the function userauth_agent(), so I did the same for my wrapper that uses it.

Reply:
OK, so there is no other choice. I see.